applications/launchpad/gui-react/src/containers/MiningContainer/MiningBoxMerged/SetupMergedWithForm.tsx

```diff
@@ -47,7 +49,10 @@ const SetupMergedWithForm = ({
       {!revealForm ? (
         <Button
           variant='primary'
-          onClick={() => setRevealForm(true)}
+          onClick={() => {
+            setRevealForm(true)
+            changeTag()
+          }}
```

Comment:
In merged mining box, we need to change the tag (`Ready to set` / `Best choice`) when we reveal the form, without changing the overall _status_ of whole box:

![image](https://user-images.githubusercontent.com/11715931/169581088-e6482377-a010-4341-ab84-5e95e6739050.png)

![image](https://user-images.githubusercontent.com/11715931/169581134-f5ad45b1-1085-46c7-a662-704519c24e24.png)